kbase-extension/static/kbase/config/staging_upload.json

```diff
@@ -64,6 +68,9 @@
     "app_info": {
         "web_upload": {
             "app_id": "kb_uploadmethods/upload_web_file"
+        },
+        "import_specification": {
```

Comment:
Needs to be here to alert the staging viewer that there's SOMETHING to use. Keys like `app_id`, etc, aren't necessary.